src/ypd/templates/CS320-ProjectMainPage.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>YCP Project Database</title>
+        <link rel="icon" href="Logo.png" type="image/png">
+        <link href="https://fonts.googleapis.com/css?family=Alfa+Slab+One&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Rokkitt&display=swap" rel="stylesheet">
+        <link href="../static/TeamProjectStyleSheet.css" rel="stylesheet" type="text/css">
+    </head>
+    
+    <body>
+        <div class="mainTitle">
+            <div class="logo">
+                <img src="../static/Logo.png" alt="Logo" width="30" height="35">
+            </div>
+            <center>YCP Project Database</center>
+                <div class="dropDown">
+                    <div class="menuTab"><center><div class="bar"></div><div class="bar"></div><div class="bar"></div></center></div>
+                        <div class="dropDown-Content">
+                            <a href="#">Link 1</a>
+                            <a href="#">Link 2</a>
+                            <a href="#">Link 3</a>
+                        </div>
+                </div>
+        </div>
+        
+        <div class="mainBody">
+            <div class="postBox">
+                <div class="dot"></div>
+                <div class="postText">
+                    test
+                </div>
+                <div class="postBar"></div>
+            </div>
+            <div class="postBox">
```

Comment:
This is ok for now, but when we have a real system up, we're going to have to use a jinja2 while statement to display posts from a list

Reply:
For the minimal working product, we're going to need to populate these divs from a Catalog object